MapboxGLAndroidSDK/src/cpp/style/layers/layer.cpp.ejs

```diff
@@ -74,15 +84,24 @@ namespace android {
 <% if (property.transition) { -%>
     jni::Local<jni::Object<TransitionOptions>> <%- camelize(type) %>Layer::get<%- camelize(property.name) %>Transition(jni::JNIEnv& env) {
         using namespace mbgl::android::conversion;
-        mbgl::style::TransitionOptions options = to<%- camelize(type) %>Layer(layer).get<%- camelize(property.name) %>Transition();
+        auto guard = layer.lock();
+        if (!layer) {
+            mbgl::Log::Error(mbgl::Event::JNI, "Failed to get layer transition options: core layer is not available");
+            return std::move(*convert<jni::Local<jni::Object<TransitionOptions>>>(env, mbgl::style::TransitionOptions{}));
+        }
+        mbgl::style::TransitionOptions options = to<%- camelize(type) %>Layer(*layer.get()).get<%- camelize(property.name) %>Transition();
         return std::move(*convert<jni::Local<jni::Object<TransitionOptions>>>(env, options));
     }
 
     void <%- camelize(type) %>Layer::set<%- camelize(property.name) %>Transition(jni::JNIEnv&, jlong duration, jlong delay) {
+        if (!layer) {
```

Comment:
`auto guard = layer.lock();` before checking `if (!layer) {`, otherwise, [pointer](https://github.com/mapbox/mapbox-gl-native-android/pull/432/files#diff-da4069f8167370d4674ea95d4e4e6bb1R104) might be null.

Reply:
Done.